frontend/apps/web/components/jobs/SchemaTable/transformer-handler.ts

```diff
@@ -74,10 +74,10 @@ function shouldIncludeSystem(
   filters: TransformerFilters
 ): boolean {
   if (
-    filters.hasDefault &&
+    !filters.hasDefault &&
     transformer.source === TransformerSource.GENERATE_DEFAULT
   ) {
-    return true;
+    return false;
```

Comment:
I don't quite understand why this was changed.
We want this block to return true if the filter says that it has a default and the source is Generate default